### ISSUE TITLE
How to create global configuration for sidekiq_mailer

### ISSUE THREAD

**Issue by coderhs** [andersondias/sidekiq_mailer#16]
Is it possible to create a global config and set all the mails to be send to queue named 'default'.

**Comment by coderhs** [comment]
@andersondias Is it possible to add an initializer and configure the gem to add to a particular queue?

**Comment by andersondias** [comment]
Nope, it's not possible right now. A Pull Request with this would be awesome :D

**Comment by coderhs** [comment]
@andersondias  : I have worked on it a bit. It is working the code can be found here https://github.com/coderhs/sidekiq_mailer/tree/configure. I have written the test for the configuration as a separate file so moved the common class to the test helper. Individually running the test file they pass, but when i run them together. The configuration that I have set for one test seems to persist when other test runs thus causing issues. I haven't been able to solve it. Will look into that issue soon and send a pull request. 

PS: Feel free to correct or point out any mistakes, in the way i have written the code.
